root/etc/e-smith/templates.metadata/etc/pam.d/cockpit

```diff
@@ -0,0 +1,3 @@
+UID="root"
+GID="root"
+PERMS="0644"
```

Comment:
I think this metadata file is not required. Why do you want to add it?

Reply:
yes this one is on my own, maybe we could be less strict :-?

Reply:
done